tests/Makefile.am

```diff
@@ -5,3 +5,5 @@ TESTS = $(check_PROGRAMS)
 
 test1_SOURCES = test1.c
 test1_LDADD = ../src/libpolycap.la
+test1_CFLAGS = @OPENMP_CFLAGS@
+test1_LDFLAGS = @OPENMP_CFLAGS@
```

Comment:
Had een issue met compilatie bij `make distcheck` (niet bij make gewoon) en zag dat in tests folder was, opnieuw die OPENMP errors. Dit toegevoegd en nu werkt het flink :)
Best trots op mezelf :D

Reply:
Nice 😄